package.json

```
@@ -36,10 +36,13 @@
   ],
   "scripts": {
     "changelog": "lerna-changelog",
-    "lint": "npm-run-all --continue-on-error --aggregate-output --parallel lint:*",
+    "lint": "npm-run-all --continue-on-error --aggregate-output --parallel \"lint:!(fix)\"",
     "lint:docs": "markdownlint \"**/*.md\"",
+    "lint:docs:fix": "markdownlint \"**/*.md\" --fix",
```

To avoid repeating the pattern and any args that get passed:
```suggestion
    "lint:docs:fix": "npm-run-all \"update:docs -- --fix\"",
```